packages/slice-machine/test/pages/slices.test.tsx

```diff
@@ -351,9 +352,14 @@ describe("slices", () => {
       fireEvent.change(nameInput, { target: { value: "FooBar" } });
     });
 
-    const submitButton = screen.getByText("Create");
+    const createSliceModal = document.querySelector<HTMLElement>(
+      '[data-cy="create-slice-modal"]'
+    );
     await act(async () => {
-      fireEvent.click(submitButton);
+      if (createSliceModal) {
```

Comment:
⚠ Because of the `if`, this test seems not useful anymore, no? Meaning, if you don't find the modal, just do nothing.

We should assess before that the element exist at least I guess